### PR TITLE
Project ID can now be found in classesInfo

### PR DESCRIPTION
**Pull request by Virgil993**
and is used to form the url

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
 import { NodeJsBundler } from "./bundlers/javascript/nodeJsBundler";
 import { NodeJsBinaryDependenciesBundler } from "./bundlers/javascript/nodeJsBinaryDepenciesBundler";
 import { getProjectConfiguration } from "./utils/configuration";
+import { REACT_APP_BASE_URL } from "./variables";
```
Comment (radu1122) [inline src/commands.ts:24]: i think this is not used

**Review by radu1122 — APPROVED** [review]
looks good

File: src/commands.ts [hunk 2]
```diff
@@ -239,6 +242,9 @@ export async function deployClasses() {
   await generateSdks(functionUrlForFilePath);
 
   reportSuccess(classesInfo, configuration);
+  
+  let projectId = classesInfo[0].projectId
```
Comment (vladiulianbogdan) [inline src/commands.ts:246]: Add this in the `reportSucess` method.

Comment (radu1122) [reply src/commands.ts:246]: it can't be added, the method its also used for local env

**Review by radu1122 — APPROVED** [review]

**Pull request merged**